pages/datasets/scaffoldviewer/_id.vue

```diff
@@ -53,7 +50,7 @@
               :url="scaffoldUrl"
               :background-toggle="backgroundToggle"
               :region="region"
-              :viewURL="viewURL"
+              :view-u-r-l="viewURL"
```

Comment:
Was this meant to happen?

Reply:
Yes, that is how camel case will be handled. All capitals replaced by hyphen and lower case of the letter.

Reply:
Unfortunately, vue props should always be camelCase during declaration but kabab-case in template, making consistence and search for a variable worse.

Reply:
mmm yeah that is unfortunate. Especially with the method of each capital gets preceded with a hyphen